### PR TITLE
fix nuget package

### PR DESCRIPTION
**Pull request by jennyf19**

### CODE REVIEW

File: tools/app-provisioning-tool/app-provisioning-tool/ms-identity-app.csproj [hunk 1]
```diff
@@ -5,12 +5,39 @@
...
+    <AssemblyOriginatorKeyFile>../../../build/MSAL.snk</AssemblyOriginatorKeyFile>
+
+    <PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
+    <!-- Copyright needs to be in the form of © not (c) to be compliant -->
+    <Title>ms-identity-app</Title>
+    <Authors>Microsoft</Authors>
+    <Company>Microsoft Corporation</Company>
+    <Product>ms-identity-app</Product>
+    <Description>
+      This package enables ASP.NET Core web apps and web APIs to use the Microsoft identity platform (formerly Azure AD v2.0).
```
Comment (jmprieur) [inline tools/app-provisioning-tool/app-provisioning-tool/ms-identity-app.csproj:22]: This package is a dotnet global tool which registers Azure AD or Azure AD B2C applications, and updates your code accordingly. For details see https://aka.ms/ms-identity-app-registration

File: tools/app-provisioning-tool/app-provisioning-tool/ms-identity-app.csproj [hunk 2]
```diff
@@ -5,12 +5,39 @@
...
+      This package is specifically used for web applications, which sign-in users, and protected web APIs, which optionally call downstream web APIs.
+    </Description>
+    <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
+    <PackageLicenseExpression>MIT</PackageLicenseExpression>
+    <PackageProjectUrl>https://github.com/AzureAD/microsoft-identity-web</PackageProjectUrl>
```
Comment (jmprieur) [inline tools/app-provisioning-tool/app-provisioning-tool/ms-identity-app.csproj:27]:
```suggestion
    <PackageProjectUrl>https://github.com/AzureAD/microsoft-identity-web/blob/master/tools/app-provisioning-tool/README.md</PackageProjectUrl>
```

File: tools/app-provisioning-tool/app-provisioning-tool/ms-identity-app.csproj [hunk 3]
```diff
@@ -5,12 +5,39 @@
...
+    <RepositoryUrl>https://github.com/AzureAD/microsoft-identity-web</RepositoryUrl>
+    <PackageReleaseNotes>The release notes are available at https://github.com/AzureAD/microsoft-identity-web/releases and the roadmap at https://github.com/AzureAD/microsoft-identity-web/wiki#roadmap </PackageReleaseNotes>
+    <PackageTags>Microsoft Identity Web;Microsoft identity platform;Microsoft.Identity.Web;.NET;ASP.NET Core;Web App;Web API;B2C;Azure Active Directory;AAD;Identity;Authentication;Authorization</PackageTags>
```
Comment (jmprieur) [inline tools/app-provisioning-tool/app-provisioning-tool/ms-identity-app.csproj:30]:
```suggestion
    <PackageTags>Microsoft Identity Web;Microsoft identity platform;Microsoft.Identity.Web;.NET;ASP.NET Core;Web App;Web API;B2C;Azure Active Directory;AAD;Identity;Authentication;Authorization;Application registration;app registration</PackageTags>
```

**Review by jmprieur — COMMENTED** [review]
LGTM. I've proposed improvements.

File: tools/app-provisioning-tool/app-provisioning-tool/ms-identity-app.csproj (hunk 1 above)
Comment (jennyf19) [reply tools/app-provisioning-tool/app-provisioning-tool/ms-identity-app.csproj:22]: right, had not even looked at this. thanks for the link.

File: tools/app-provisioning-tool/app-provisioning-tool/ms-identity-app.csproj (hunk 2 above)
Comment (jennyf19) [reply tools/app-provisioning-tool/app-provisioning-tool/ms-identity-app.csproj]: fixed.

File: tools/app-provisioning-tool/app-provisioning-tool/ms-identity-app.csproj (hunk 3 above)
Comment (jennyf19) [reply tools/app-provisioning-tool/app-provisioning-tool/ms-identity-app.csproj]: fixed.

File: tools/app-provisioning-tool/app-provisioning-tool/ms-identity-app.csproj (hunk 1 above)
Comment (jennyf19) [reply tools/app-provisioning-tool/app-provisioning-tool/ms-identity-app.csproj]: made a few more changes here

**Pull request merged**